### PR TITLE
Fix rendering of cfp PR body

### PR DESCRIPTION
**Pull request by isuruf**

### CODE REVIEW

File: conda_forge_tick/migrators/migration_yaml.py [hunk 1]
```diff
@@ -367,13 +367,12 @@ def pr_body(self, feedstock_ctx: "FeedstockContext") -> str:
...
             "Checklist:\n"
-            "- [ ]. The new version is a stable supported pin. \n"
-            "- [ ]. I checked that the ABI changed from {current_pin} to "
+            "- [ ] The new version is a stable supported pin. \n"
+            "- [ ] I checked that the ABI changed from {current_pin} to "
             "{new_pin_version}. \n"
             "\n"
             "**Please note that if you close this PR we presume that "
             "the new pin has been rejected.\n\n"
-            "@conda-forge-admin please ping {feedstock_name}\n"
```
Comment (beckermr) [inline conda_forge_tick/migrators/migration_yaml.py:376]: @CJ-Wright I think you added this here. You ok removing it and having the bot generate the pinning PRs without pinging the feedstock maintainers?

Comment (isuruf) [reply conda_forge_tick/migrators/migration_yaml.py:376]: They already get review requested, so there's no point.

Comment (beckermr) [reply conda_forge_tick/migrators/migration_yaml.py:376]: There is a request for review for the core team on the pinnings repo. This line pings the feedstock maintenance team about changing the pin.

Comment (isuruf) [reply conda_forge_tick/migrators/migration_yaml.py]: Oops. Fixed

Comment (beckermr) [reply conda_forge_tick/migrators/migration_yaml.py]: thank you!

**Pull request merged**